tachyon/math/polynomials/multivariate/multilinear_dense_evaluations.h

```diff
@@ -144,6 +144,41 @@ class MultilinearDenseEvaluations {
         std::vector<F>(poly.begin(), poly.begin() + (size_t{1} << (n - k))));
   }
 
+  // Fix k variables out of n variables in place, where k is
+  // |partial_point.size()| and n is |Degree()|.
+  void FixVariablesInPlace(const Point& partial_point) {
```

Comment:
Why did you make this?

Reply:
To prevent excessive copies with `FixVariables()`